javatests/arcs/core/crdt/CrdtSetTest.kt

```diff
@@ -242,7 +265,7 @@ class CrdtSetTest {
                 )
             )
         assertThat(fastForward.removed)
-            .containsExactly(Reference("removed by alice"))
+            .containsExactly(Reference("removed by alice"), Reference("removed by bob"))
```

Comment:
bob already knows about the "removed by bob" operation. There's no need to send it back again.

(`fastForward` here is `otherChange`, i.e. to send to bob)